docs/reboot.md

```diff
@@ -43,10 +43,10 @@ The queue is processed by CKE as follows:
 3. For the first entry in the reboot queue, do the following steps.
    1. Update the entry status to `rebooting`.
    2. Cordon the nodes in the entry.
-   3. Call the eviction API for Pods running on the target nodes.  DaemonSet-managed Pods are ignored.  If pods not in the `protected-namespaces` fail to be evicted, they are deleted instead.
-   4. Wait for the deletion of the Pods.  If this step exceeds a time limit specified in the cluster configuration, this operation is canceled.
-   5. Reboot the nodes using `.reboot.command` in the cluster configuration.  In this step, all the nodes are rebooted simultaneously.  If this step exceeds a time limit specified in the cluster configuration, give up waiting.
-   6. Uncordon the successfully rebooted nodes.
+   3. Call the eviction API for Pods running on the target nodes.  DaemonSet-managed Pods are ignored.  If pods not in the `protected_namespaces` fail to be evicted, they are deleted instead.
+   4. Wait for the deletion of the Pods.  If this step exceeds a deadline specified in the cluster configuration, the operation is aborted and the queue entry is left as is.
+   5. Reboot the nodes using `.reboot.command` in the cluster configuration.  In this step, all the nodes are rebooted simultaneously.  If this step exceeds a deadline specified in the cluster configuration, give up waiting.
```

Comment:
5. Reboot the nodes using `.reboot.command` in the cluster configuration.  In this step, all the nodes are rebooted simultaneously.  If some of the nodes won't get back ready within the deadline specified in the cluster configuration, CKE gives up waiting for them (no error).